src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java

```diff
@@ -247,6 +248,12 @@ protected String[] customCommandLineValidation() {
                 errors.add("Parameters 'project-id', 'dataset-id', 'call-set-identifier', 'wdl-step', 'wdl-call', and 'shardIdentifier' must be set if 'cost-observability-tablename' is set.");
             }
         }
+        if (truthSensitivitySNPThreshold < 0.0 || truthSensitivitySNPThreshold > 1.0) {
+            errors.add("Parameter 'snps-truth-sensitivity-filter-level' MUST be between 0.0 and 1.0'");
```

Comment:
should we print out what the number actually is?

Reply:
Done